docsets/Salesforce_Lightning/docset.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Salesforce Lightning",
+    "version": "40.0",
+    "archive": "Salesforce_Lightning.tgz",
```

Comment:
There's no `Salesforce_Lightning.tgz` in the main folder. You need to add it. Also remove `Sample_Docset.tgz`.